src/OSPSuite.Core/Domain/Mappers/ParameterBuilderToParameterMapper.cs

```diff
@@ -19,6 +19,8 @@ public ParameterBuilderToParameterMapper(ICloneManagerForModel cloneManagerForMo
       public IParameter MapFrom(IParameter parameterBuilder, IBuildConfiguration buildConfiguration)
       {
          var parameter = _cloneManagerForModel.Clone(parameterBuilder);
+         //We reset the container criteria explicitly in the model instance 
+         parameter.ContainerCriteria = null;
```

Comment:
Parameter that are instanced in model do not need a container criteria (they are already created). So we remove